doc/source/development.rst

```diff
@@ -10,14 +10,21 @@ Branches
 * The ``master`` branch is stable.  Major changes should be performed
   elsewhere.
 
+Managing the Changelog
+======================
+
+Please see `changelog/README.rst` for instructions.
```

Comment:
You should be able to do
```suggestion
Please see `<changelog/README.rst>`_ for instructions.
```

Ref: https://thomas-cokelaer.info/tutorials/sphinx/rest_syntax.html#external-links